### ISSUE TITLE
Feature Announcement: add a custom layout for the My Site Dashboard announcement

### ISSUE THREAD

**Issue by leandroalonso** [wordpress-mobile/WordPress-iOS#18209]
We want to make a feature announcement with a slightly different template:

![custom](https://user-images.githubusercontent.com/7040243/160006273-86049ba3-de6c-4f23-9f12-b22f73467863.png)

I explored the current feature announcement tool to use the code and just change the template for this specific announcement. My exploration can be found here: https://gist.github.com/leandroalonso/57801f0bb7d0ec6ab0f005c61491e7b6

Basically, we're able to use all the existing logic from the feature announcement (including setting up the text content on the web tool) and just build our own custom view on top of the existing one.

**Comment by hassaanelgarem** [comment]
@leandroalonso @momo-ozawa 

Since the web tool is not built with this design in mind, in order to use it without hardcoding anything we'll have to come up with a workaround. What I have in mind is the following:
- We'll add a feature with "Dashboard: your site at a glance" as the title and "NEW" icon as the icon and no subtitle
- Add two other features with an empty title and image as the icon and image description as the subtitle
- From our side, we'll look for the feature without a subtitle to use as the heading, and the remaining features will be used to populate a tableview.

What do you think? Did you have something else in mind?

**Comment by osullivanchris** [comment]
Here's a final design and specs. Just asking @leandroalonso a clarification on the image assets on DM, then I'll attach those

![home feature announcement](https://user-images.githubusercontent.com/50576199/160139128-3cf3221e-895d-470e-9903-1d41e4a026f2.png)

dark mode should look something like this
![darkmode](https://user-images.githubusercontent.com/50576199/160142171-ca93ac8d-7eb4-4112-86f3-9ea87d966d19.jpg)

**Comment by leandroalonso** [comment]
@hassaanelgarem sounds like a plan to me. One thing though, about:

> From our side, we'll look for the feature without a subtitle to use as the heading, and the remaining features will be used to populate a tableview.

I wonder if it wouldn't be better to check for the `appVersionName` that is returned by the API to show the custom view. Up to you anyway!

**Comment by osullivanchris** [comment]
Here are some image assets. If we need different format, size, etc, happy to produce

[my site feature announcement images.zip](https://github.com/wordpress-mobile/WordPress-iOS/files/8351414/my.site.feature.announcement.images.zip)

**Heading**
My Site Home

**Body 1**
My Site has a new home tab showing you up to date highlights from your site.

**Body 2**
Your menu hasn’t gone anywhere. You can switch between home and menu any time.

**Comment by hassaanelgarem** [comment]
@leandroalonso 
> I wonder if it wouldn't be better to check for the appVersionName that is returned by the API to show the custom view.

Yes, I'm going to use the `appVersionName` to decide on which view to use. I meant I'll look for the feature without subtitle after that, to decide which feature to use as the title and which features to use for populating the tableview 👍

**Comment by hassaanelgarem** [comment]
@osullivanchris Can you please provide the following specs 🙏 
- Image background color
- Image corner radius
- Button title font
- The "NEW!" disclaimer background color

**Comment by osullivanchris** [comment]
@hassaanelgarem hey thanks for keeping an eye on those details. Here's the specs:

- Image background color: from Apple's components lets use System Background: "Quartinary Fill color". Comes out as F4F4F5 in light mode and 747480 in dark mode
- Image corner radius 10pt
- Button title font: can we pull from other buttons we're using in onboarding? I don't want to create a new style. But not 100% sure what they're using. If its not a component we can reuse, lets at least copy the style.
- "New!" background color. Lets use from [color studio](https://color-studio.blog/) Pink 40 #e34c84 in light mode and Pink 50 #c9356e in dark mode
